src/tablist.js

```diff
@@ -307,6 +319,34 @@ SideTabList.prototype = {
     Math.max(0, dropTabPos);
     browser.tabs.move(tabId, { index: newPos });
   },
+  async onScroll(e) {
+    if (!this.scrollTabs) {
+      return;
+    }
+    e.preventDefault();
+    // Gets tabs in window, then derives active tab
+    let tabs = await browser.tabs.query({currentWindow: true});
```

Comment:
Thanks! Is there an up-to-date array of tabs so we don't even have to do the query?